src/atoms/index.ts

```diff
@@ -18,7 +22,39 @@ export const favoriteCoinListState = atom<Market[]>({
   effects: [storageEffect(RECOIL_KEY.favoriteCoinList, 'localStorage')],
 });
 
+export const favoriteMarketsState = selector({
+  key: RECOIL_KEY.favoriteMarkets,
+  get: ({get}) => {
+    return get(favoriteCoinListState).map(x => x.market);
+  },
+});
+
+export const isFavoriteMarketState = selectorFamily<boolean, string>({
+  key: RECOIL_KEY.isFavoriteMarket,
+  get:
+    market =>
+    ({get}) => {
+      return get(favoriteMarketsState).includes(market);
+    },
+});
+
 export const selectedCoinState = atom<Market[]>({
   key: RECOIL_KEY.selectedCoin,
   default: [],
 });
+
+export const selectedMarketState = selector({
+  key: RECOIL_KEY.selectedMarket,
+  get: ({get}) => {
+    return get(selectedCoinState).map(x => x.market);
+  },
+});
+
+export const isSelectedCoinState = selectorFamily<boolean, string>({
+  key: RECOIL_KEY.isSelectedCoin,
+  get:
+    market =>
+    ({get}) => {
+      return get(selectedMarketState)[0] === market;
+    },
+});
```

Comment:
```suggestion
export const isSelectedCoinState = selectorFamily<boolean, string>({
  key: RECOIL_KEY.isSelectedCoin,
  get:
    market =>
    ({get}) => {
      return get(selectedCoinState)?.market === market;
    },
});
```

이 부분도 마찬가지! 앞서 말한대로 굳이 selector로 따로 만들 필요가 없다고 생각해요 ㅎㅎ  
이 코드는 제가 위에서 `selectedCoinState`를 `Market | null`로 지정해서 이렇게 코드가 나오네요~  
배열은 해당 state의 이름과는 어울리지 않다고 생각합니다 ㅎㅎ

Reply:
음 이러면 더 깔끔해지네요!